embulk-api/src/main/java/org/embulk/spi/PageReader.java

```diff
@@ -181,19 +182,57 @@ public Instant getTimestampInstant(final int columnIndex) {
     }
 
     /**
+     * Returns a JSON value at the specified column in the {@code msgpack-java} representation.
+     *
+     * @param column  the column to get the JSON value
+     * @return the JSON value in the {@code msgpack-java} representation
+     * @deprecated Use {@link #getJsonValue(int)} instead.
```

Comment:
This is the description `getJson(Column column)` method.
So, `@deprecated Use {@link #getJsonValue(Column)} instead` ?

`int` -> `org.embulk.spi.Column`?

Reply:
Good catch, too!